__tests__/e2e/utils/settings.ts

```diff
@@ -5,17 +5,28 @@ import { E2eSettingsOptions } from '../constants/settings'
 
 export const getEncryptSettings = (
   custom?: Partial<E2eSettingsOptions>,
+  isSubmitterIdCollectionEnabled = false,
```

Comment:
It looks like `E2eSettingsOptions` already has `isSubmitterIdCollectionEnabled` props. Could this be extracted from `custom?: Partial<E2eSettingsOptions>`?

Reply:
Reason for asking is because I didn't see any usage of `isSubmitterIdCollectionEnabled: true`

Reply:
fixed!